pkg/cmd/tests/tests_run.go

```diff
@@ -113,18 +82,32 @@ func NewRunOptions(streams genericclioptions.IOStreams) *RunOptions {
 		ParallelShard:         0,
 		ParallelServerAddress: "",
 		ParallelLogLevel:      0,
+
+		testSuites: testSuites,
 	}
 }
 
-func NewRunCommand(streams genericclioptions.IOStreams) *cobra.Command {
-	o := NewRunOptions(streams)
+var AllTestSuite = &ginkgotest.TestSuite{
```

Comment:
this feels a bit off to be defined in a different file then the Suites

Reply:
removed